includes/class-freemius.php

```diff
@@ -9789,6 +9845,57 @@ private function get_install_diff_for_api( $site, $install, $override = array()
             return $diff;
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since  2.5.1
+         *
+         * @param FS_Site  $site
+         * @param string[] $params
+         */
+        function maybe_send_clone_update( $site = null, $params = array(), $initial_update = false ) {

```

Comment:
@fajardoleo
* Let's split it into 2 methods
* Let's remove the `$this->is_tracking_allowed()` check (the intent of the resolution is there when attempting to resolve regardless of the opt-in permissions)